### ISSUE TITLE
I don't see this device on my network

### ISSUE THREAD

**Issue by Pavel20481962** [Willtech/NTP-Server-for-ESP8266#1]
hi, I installed your firmware on ESP8266. Replace YourWiFiAPN and YourWiFiPassword with the SSID and password of your WiFi network. but I don't see this device on my network.

**Comment by Pavel20481962** [comment]
mistake const int D0 = 14; // SDL

**Comment by Willtech** [comment]
Yes you must configure for your board. Many ESP8266 device work with the similar configuration as pictured. Unfortunately because the published pictures of the built work may be used for a qualification it would disqualify if you have ever looked at the Internet unless the testing Institute can verify you have never looked. The output on Linux looks like:

```BASH
$ntpdate -q {localNTPip}
2024-10-20 16:29:52.000000 (+1100) -0.364306 +/- 0.047946 10.20.10.125 s1 no-leap
```

You have not asked for a hint?

**Comment by Willtech** [comment]
You can try this to scan for the NTP Server. [Willtech/BASH-NTP-Date-Log](https://github.com/Willtech/BASH-NTP-Date-Log)

If you before you first run use `echo {localIP:192.168.1.0} > bashntpdate.log` it will begin with on your local network address.

**Comment by Pavel20481962** [comment]
Спасибо всё работает хорошо.----------------
                        Кому: Willtech/NTP-Server-for-ESP8266 ***@***.***);
                        Копия: Author ***@***.***);
                        Тема: [Willtech/NTP-Server-for-ESP8266] I don't see this device on my network (Issue #1);
                        25.04.2025, 20:18, "Willtech" ***@***.***>:
                        
Willtech left a comment (Willtech/NTP-Server-for-ESP8266#1)
You can try this to scan for the NTP Server. Willtech/BASH-NTP-Date-Log
If you before you first run use echo {localIP:192.168.1.0} > bashntpdate.log it will begin with on your local network address.
